### PR TITLE
Font Library Preview: fix quoting of fontFamily property

### PR DESCRIPTION
**Pull request by creativecoder**
## What?

- Fix font face previews in the font library, which may not show up when the font family contains a space, e.g. Alegreya Sans
- Fix a console error when loading the preview

## Why?

To see previews for installed font faces.

## How?

Make sure font family names are quoted only once.

## Testing Instructions

- Install a font in the Font Library that has a name with a space, like "Alegreya Sans"
- Without this branch
    - The font face previews (after you click on the font name to see its variants) will not preview correctly
    - You may see a console error: `Uncaught (in promise) DOMException: An invalid or illegal string was specified - loadFontFaceInBrowser index.js:104`
- With this branch
    - Previews show correctly
    - There is not console error

### CODE REVIEW

File: packages/edit-site/src/components/global-styles/font-library-modal/utils/index.js [hunk 1]
```diff
@@ -93,10 +93,11 @@ export async function loadFontFaceInBrowser( fontFace, source, addTo = 'all' ) {
...
 	} else if ( source instanceof File ) {
 		dataSource = await source.arrayBuffer();
+	} else {
+		return;
 	}
 
-	// eslint-disable-next-line no-undef
-	const newFont = new FontFace( fontFace.fontFamily, dataSource, {
+	const newFont = new window.FontFace( fontFace.fontFamily, dataSource, {
 		style: fontFace.fontStyle,
```
Comment (creativecoder) [inline packages/edit-site/src/components/global-styles/font-library-modal/utils/index.js]: This changes are unrelated--just fixing a couple of red squiggles in my editor.

File: packages/edit-site/src/components/global-styles/font-library-modal/utils/preview-styles.js [hunk 2]
```diff
@@ -35,9 +35,9 @@ export function formatFontFamily( input ) {
 		.split( ',' )
 		.map( ( font ) => {
 			font = font.trim(); // Remove any leading or trailing white spaces
-			// If the font doesn't have single quotes and contains a space, then add single quotes around it
-			if ( ! font.startsWith( "'" ) && font.indexOf( ' ' ) !== -1 ) {
-				return `'${ font }'`;
+			// If the font doesn't have quotes and contains a space, then wrap in quotes.
+			if ( ! font.startsWith( '"\'' ) && font.indexOf( ' ' ) !== -1 ) {
+				return `"${ font }"`;
 			}
```
Comment (creativecoder) [inline packages/edit-site/src/components/global-styles/font-library-modal/utils/preview-styles.js:45]: Check for both `"` and `'` before adding quotes, since either quote type is valid for wrapping font family names.

**Comment by creativecoder** [comment]
Some tests are failing--I'll work on fixing those.

**Review by mikachan — APPROVED** [review]
Works well for me and tests are passing locally 🚢

**Comment by creativecoder** [comment]
I've updated the PR and the description--this now fixes a console error that was happening, as well.

**Comment by creativecoder** [comment]
> Works well for me and tests are passing locally 🚢

You must have reviewed it just after I updated it 😅

**Comment by mikachan** [comment]
> You must have reviewed it just after I updated it 😅

I was in the middle of writing a comment about the tests failing and then I saw your update 😆
